### PR TITLE
Add *Arr Suite category for Website

### PR DESCRIPTION
**Pull request by MickLesk**
## Description
https://github.com/community-scripts/ProxmoxVE/discussions/273

Created PR for *Arr Suite.

The only thing I was really unsure about was homarr. Or do we want to leave homarr in the dashboard area?

## Type of change
Please check the relevant option(s):

- [ ] Bug fix (non-breaking change that resolves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (a fix or feature that would cause existing functionality to change unexpectedly)
- [ ] New script (a fully functional and thoroughly tested script or set of scripts.)

## Prerequisites
The following efforts must be made for the PR to be considered. Please check when completed:
- [x] Self-review performed (I have reviewed my code, ensuring it follows established patterns and conventions)
- [x] Testing performed (I have tested my changes, ensuring everything works as expected)
- [x] Documentation updated (I have updated any relevant documentation)

### CODE REVIEW

**Review by bramsuurdje — DISMISSED** [review]

**Comment by havardthom** [comment]
There is also `autobrr`

File: json/metadata.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
         {"name": "Operating System", "id": 9, "sort_order": 14.0},
         {"name": "Server - Networking", "id": 11, "sort_order": 15.0},
         {"name": "TurnKey", "id": 10, "sort_order": 16.0},
-        {"name": "Zigbee - Zwave", "id": 6, "sort_order": 17.0},
+        {"name": "Matter/Zigbee/ZWave", "id": 6, "sort_order": 17.0},
+        {"name": "*arr Suite", "id": 18, "sort_order": 18.0},
```
[the remark this answers is not in the document]
Comment (BramSuurdje) [reply json/metadata.json:21]: we sort it alphabetically on build. no need for the sort_order

[the remark this answers is not in the document]
Comment (MickLesk) [reply json/metadata.json:21]: I think *arr Suite sounds better :D

[the remark this answers is not in the document]
Comment (MickLesk) [reply json/metadata.json:21]: > I believe I changed that when moving to json assets. `sort_order` in metadata is the correct order now

You can switch the branch here to arr Suite branch and then you can sort it

(Its an Project Branch)

[the remark this answers is not in the document]
Comment (havardthom) [reply json/metadata.json:21]: from https://wiki.servarr.com/
> Collectively they are referred to as "*Arr", "*Arrs", "Starr", or "Starrs"

maybe `*Arr Suite`? or we can just keep as it is now, but i think it should be moved between adblocker and automation

[the remark this answers is not in the document]
Comment (havardthom) [reply json/metadata.json:21]: I like 3rd

[the remark this answers is not in the document]
Comment (MickLesk) [reply json/metadata.json:21]: Then 2nd, because all These Scripts named as *arr Scripts

**Review by havardthom — APPROVED** [review]

**Review by bramsuurdje — APPROVED** [review]

**Comment by arkdev1** [comment]
Hi @MickLesk, the '*' in front of *Arr Suite is correct?

**Comment by MickLesk** [comment]
> Hi @MickLesk, the '*' in front of *Arr Suite is correct?

yep

**Comment by arkdev1** [comment]
> > Hi @MickLesk, the '*' in front of *Arr Suite is correct?
> 
> yep

I apologize, I thought it was a typo.

image

**Pull request merged**